psl/parser-database/src/lib.rs

```diff
@@ -131,18 +153,30 @@ impl ParserDatabase {
         relations::infer_relations(&mut ctx);
 
         ParserDatabase {
-            ast,
-            file,
+            asts,
             interner,
             names,
             types,
             relations,
         }
     }
 
-    /// The parsed AST.
-    pub fn ast(&self) -> &ast::SchemaAst {
-        &self.ast
+    /// The parsed AST. This methods asserts that there is a single prisma schema file. As
+    /// multi-file schemas are implemented, calls to this methods should be replaced with
+    /// `ParserDatabase::ast()` and `ParserDatabase::iter_asts()`.
+    pub fn ast_assert_single(&self) -> &ast::SchemaAst {
+        assert_eq!(self.asts.0.len(), 1);
+        &self.asts.0.first().unwrap().2
```

Comment:
Do we really need all this? Both `assert_eq()` and `.unwrap()` use the `panic!` macro under the hood. I'd recommend replacing `assert_eq` with `debug_assert_eq`: this will allow us to test the assertion at test time, without "wasting time" at runtime in optimised production builds.

Reply:
I think the idea is, `_assert_single` methods on the `ParserDatabase` will go away once the feature is done. So, IMHO, it is fine to panic for now.